### ISSUE TITLE
LogicUtils: investigate if we can use `$ignore` placeholder to ignore any levels

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/carina#1639]
For example below use case generate an exception as we have different url levels:
```
LogicUtils.isURLEqual("https://www.mysite.com/$ignore", "https://www.mysite.com/welcome/stream");
```

let's investigate if we can use `$ignore` for any sub levels to avoid such notation which works now:
```
LogicUtils.isURLEqual("https://www.mysite.com/$ignore/$ignore", "https://www.mysite.com/welcome/stream");
```

**Comment by azarouski** [comment]
Issue was fixed
https://github.com/zebrunner/carina/pull/1670

**Comment by vdelendik** [comment]
merged, changes might be tested using `7.3.20-SNAPSHOT`
